src/Core/Domain/ApiAccess/Exception/ApiAccessConstraintException.php

```diff
@@ -43,6 +43,8 @@ class ApiAccessConstraintException extends ApiAccessException
     public const CLIENT_ID_TOO_LARGE = 8;
     public const CLIENT_NAME_TOO_LARGE = 9;
     public const DESCRIPTION_TOO_LARGE = 10;
+    public const INVALID_SCOPES = 11;
```

Comment:
Is this case ever used ?

Reply:
Yes it is, in the same other code your commented in the setter